frontend/src/composables/useShootContext/index.js

```diff
@@ -124,10 +133,29 @@ export function createShootContextComposable (options = {}) {
     return normalizeManifest(object)
   })
 
+  function setExtension (shootManifest, type, extension) {
```

Comment:
The function should be declared outsine of the composable scope.

Reply:
moved outside of composable scope